benchmarks/python/test_broadcast_add_fwd.py

```diff
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+import pytest
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype, clear_cuda_cache
+from .core import run_benchmark, clear_dynamo_cache
+import torch
+from .global_params import generate_input_sizes, FLOAT_DTYPES, PROMOTE_DTYPES
+
+
+def bcast_add_fusion(
+    fd: FusionDefinition,
+    dtype: DataType,
+    bcast_axis: int,
+) -> None:
+    T0 = fd.define_tensor(
+        shape=[-1], contiguity=[True], dtype=dtype, is_cpu=False, stride_order=[0]
+    )
+    T1 = fd.define_tensor(
+        shape=[-1, -1],
+        contiguity=[True, True],
+        dtype=dtype,
+        is_cpu=False,
+        stride_order=[1, 0],
+    )
+
+    bcast_shape = [T1.size(0), T1.size(1)]
+    bcast_shape[bcast_axis] = 1
+
+    T2 = fd.ops.broadcast_in_dim(T0, shape=bcast_shape, broadcast_dims=[1 - bcast_axis])
+    T3 = fd.ops.broadcast_in_dim(T2, shape=T1.shape(), broadcast_dims=[0, 1])
```

Comment:
Why are there two broadcast ops? Is the second one necessary?

Reply:
No, we can combine them into a separate op.
But I kept the fusion definition consistent with what Thunder generated.

For outer broadcast, if the fn is `x + bias`, then Thunder only produces one bcast op.
The use of `unsqueeze` generates 2 broadcast ops.

Reply:
OK, can you just leave a comment about that?

Reply:
Added a comment.